src/preamble.js

```diff
@@ -1104,9 +1110,9 @@ function createWasm() {
 #endif
 #if MODULARIZE
   // If instantiation fails, reject the module ready promise.
-  instantiateAsync().catch(readyPromiseReject);
+  instantiateAsync(wasmBinary, wasmBinaryFile, info, receiveInstantiationResult).catch(readyPromiseReject);
 #else
-  instantiateAsync();
+  instantiateAsync(wasmBinary, wasmBinaryFile, info, receiveInstantiationResult);
```

Comment:
Please check if there is a code size downside to this PR. Probably not after closure, but good to make sure. `./test/runner other.test*code*size* other.test*metadce* --rebaseline` should update the relevant files and then the diff would show any difference.

Reply:
Hmm, it's annoying that there's a fairly consistent but small regression on most of those. I guess from adding explicit params instead of using outer scopes. I don't have a good idea for avoiding that, but maybe someone else does? If not it seems acceptable.